packages/kbn-esql-validation-autocomplete/src/validation/__tests__/validation.command.from.test.ts

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
I moved all `FROM` source command tests to this separate file and grouped them using `describe(` into meaningful groups. The `METRICS` command works very much like the `FROM` command, so I needed to see all these tests to replicate them for `METRICS`.

Reply:
All the tested queries should be written into the `packages/kbn-esql-validation-autocomplete/src/validation/esql_validation_meta_tests.json` file.
Moving the tests is fine as long as the JSON is kept in sync.